Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/PBR/Lighting/LightingData.azsli

```diff
@@ -29,15 +29,6 @@ class LightingData
     // Direction light shadow coordinates
     float3 shadowCoords[ViewSrg::MaxCascadeCount];
 
-    // (N . L) to accept below (N . L = 0) in scattering through thin objects
-    float transmissionNdLBias; 
-    
-    // Shrink (absolute) offset towards the normal opposite direction to ensure correct shadow map projection
-    float shrinkFactor;
-
-    // Attenuation applied to hide artifacts due to low-res shadow maps 
-    float distanceAttenuation;
```

Comment:
We should preserve these comments. Please copy them to the corresponding TransmissionSurfaceData functions.

Reply:
Done!